integration_tests/two_sided_pay_as_clear_market.feature

```diff
@@ -27,8 +27,14 @@ Feature: Two sided pay_as_clear market tests
      When we run the d3a simulation on console with two_sided_pay_as_clear.default_2a for 2 hrs
      Then there are files with offers, bids & market_clearing_rate for every area
 
-   Scenario: Supply Demand Curve
+  Scenario: Supply Demand Curve
      Given we have a scenario named two_sided_pay_as_clear/test_clearing_energy
      And d3a is installed
      When we run the d3a simulation on console with two_sided_pay_as_clear.test_clearing_energy for 24 hrs
      Then the export functionality of supply/demand curve is tested
+
+  Scenario: No unmatched load
```

Comment:
Can you please also  prepend the ticket number here?

Reply:
I would also find it better to move this on the jira_issues_tests feature file.